filters.txt

```diff
@@ -1047,14 +1047,18 @@ minecrafak.cz###sidebar
 mladypodnikatel.cz##.scroll_banner
 mobilizujeme.cz###leaderbanner_space
 mobilizujeme.cz,videacesky.cz##.fullbanner
+mobilmania.cz##div.sky-wrapper
 mobilmania.cz##.not4bbtext + a
 mobilmania.cz##[id*="3.1.1"]
-mobilmania.cz##div:-abp-has(> #HalfPageAd-1)
-mobilmania.cz##div:-abp-has(> #HalfPageAd-2)
-mobilmania.cz##div:-abp-has(> #Leaderboard-1)
 mobilmania.cz##iframe[src*="alza"]
 mobilmania.cz##noscript + script + div
 mobilmania.cz##table#prolinkhdr-v3 + div:not(.cs-cnt-wrapper):not(#mainwrap)
+mobilmania.cz###content-lead
+mobilmania.cz,zive.cz###Leader-1
+mobilmania.cz,zive.cz###bgTop-iFrame
+mobilmania.cz,zive.cz##.redirectArea
+mobilmania.cz,zive.cz##div[id$="_overide"]
+mobilmania.cz,zive.cz##div[class$="_chaoz"]
```

Comment:
Also where do you see all these elements? I don't get any of these.

Reply:
![image](https://user-images.githubusercontent.com/7593516/48306946-f244d680-e542-11e8-805c-330bd064cd58.png)